src/components/LoginForm.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from "react";
+
+function LoginForm({ Login, error }) {
+  const [details, setDetails] = useState({ email: "", password: "" });
+
+  const submitHandler = (e) => {
+    e.preventDefault();
+
+    Login(details);
+  };
+
+  return (
+    <form onSubmit={submitHandler}>
+      <div className="form-inner">
+        <h2>Login</h2>
+        {error !== "" ? <div className="error">{error}</div> : ""}
+        <div className="form-group">
+          <label html="email">Email:</label>
+          <input
+            type="text"
+            name="email"
+            id="email"
```

Comment:
Are these supposed to be unique? Fine for now obviously, but eventually will probably need to hash something from the field to generate an ID, or maybe some better method.